package-lock.json

```diff
@@ -0,0 +1,7239 @@
+{
```

Comment:
We’ll want to remove this—we use [pnpm](https://pnpm.io) to manage this package, not `npm`.